README.md

```diff
@@ -25,20 +25,54 @@ datasetinsights download \
 ```
 [Programmatically](https://datasetinsights.readthedocs.io/en/latest/datasetinsights.io.downloader.html#module-datasetinsights.io.downloader.gcs_downloader)
 
-```python3
-
-from datasetinsights.io.downloader import UnitySimulationDownloader,
-GCSDatasetDownloader, HTTPDatasetDownloader
+UnitySimulationDownloader downloads a dataset from Unity Simulation.
 
+```python3
+from datasetinsights.io.downloader import UnitySimulationDownloader
 downloader = UnitySimulationDownloader(access_token=access_token)
 downloader.download(source_uri=source_uri, output=data_root)
-
+```
+GCSDatasetDownloader downloads a dataset from GCS location.
+```python3
+from datasetinsights.io.downloader import GCSDatasetDownloader
 downloader = GCSDatasetDownloader()
 downloader.download(source_uri=source_uri, output=data_root)
-
+```
+HTTPDatasetDownloader downloads a dataset from any HTTP(S) location.
+```python3
+from datasetinsights.io.downloader import HTTPDatasetDownloader
 downloader = HTTPDatasetDownloader()
 downloader.download(source_uri=source_uri, output=data_root)
+```
+### Dataset Explore
+You can explore the dataset [schema](https://datasetinsights.readthedocs.io/en/latest/Synthetic_Dataset_Schema.html#synthetic-dataset-schema) by using following API:
 
+[Unity Perception](https://datasetinsights.readthedocs.io/en/latest/datasetinsights.datasets.unity_perception.html#datasetinsights-datasets-unity-perception)
+
+AnnotationDefinitions and MetricDefinitions loads synthetic dataset definition tables and return a dictionary containing the definitions.
+
+```python3
+from datasetinsights.datasets.unity_perception import AnnotationDefinitions,
+MetricDefinitions
+annotation_def = AnnotationDefinitions(data_root="/data", version="my_schema_version")
+definition_dict = annotation_def.get_definition(def_id="my_definition_id")
+
+metric_def = MetricDefinitions(data_root="/data", version="my_schema_version")
```

Comment:
The path here should be a follow after the dataset was downloaded. I suggest using a consistent path as in the download example.